src/components/projects/A32NX/ExtendedFeatures.tsx

```diff
@@ -1,89 +1,126 @@
+
+            <Feature key="Accurate System Functionality">
+                <IconSquareCheck size={40} stroke={1.5} color="teal" />
                 <FeatureTitle>
-                    Electronic Flight Bag
+                    Accurate System Functionality
                 </FeatureTitle>
                 <FeatureBody>
-                    The EFB is an integrated tablet with a handy toolkit, a settings page, performance tools, and more.
+                    The experimental version of the A32NX includes full rewrites of the default autopilot, Fly-by-wire system and much more.
                 </FeatureBody>
             </Feature>
 
             <Feature key="Custom Soundpack">
+                <IconHeadset size={40} stroke={1.5} color="teal" />
                 <FeatureTitle>
                     Custom Soundpack
                 </FeatureTitle>
                 <FeatureBody>
-                    Enjoy sounds like never before, the A32NX includes a fully custom and highly accurate soundpack.
+                    The A32NX includes a fully custom and highly accurate sound-pack to enhance that little bit of realism.
                 </FeatureBody>
             </Feature>
 
-            <Feature key="Simbrief Integration">
+            <Feature key="Constant Additions">
+                <IconReplace size={40} stroke={1.5} color="teal" />
                 <FeatureTitle>
-                    Simbrief Integration
+                    Constant Additions
                 </FeatureTitle>
                 <FeatureBody>
-                    Make flight plan entry a breeze with the SimBrief integration, all it takes is one click and you are ready to go!
+                    The A32NX development branches are updated almost daily with brand new features you can test right away and provide feedback on!
                 </FeatureBody>
             </Feature>
 
-            <Feature key="Product Installer">
+            <Feature key="Documentation">
+                <IconFileText size={40} stroke={1.5} color="teal" />
                 <FeatureTitle>
-                    Product Installer
+                    Documentation
                 </FeatureTitle>
                 <FeatureBody>
-                    FlyByWire has a custom product installer, which makes the task of installing all FlyByWire products hassle free.
+                    The FlyByWire team has created several tutorials and documents to make learning the A320 quickly and accurately possible for anyone!
+                </FeatureBody>
+            </Feature>
+
+            <Feature key="Electronic Flight Bag">
+                <IconDeviceTablet size={40} stroke={1.5} color="teal" />
+                <FeatureTitle>
+                    Electronic Flight Bag
+                </FeatureTitle>
+                <FeatureBody>
+                    The FlyByWire team has created a custom EFB that comes with a handy toolkit, a settings page, performance tools, and much more!
```

Comment:
```suggestion
                    The A32NX comes packed with a custom EFB that comes with a handy toolkit, settings page, performance tools, and much more!
```
Grammar issue with comma separated and EFB was not exclusively made by the "team" so I would recommend this instead :)

Reply:
comes...comes seems a little clumsy.
```suggestion
                    The A32NX comes packed with a custom EFB including a handy toolkit, settings page, performance tools, and much more!
```